File: solutions/js/base/is_prime.js
// Costome function to check if a number is prime
const IsPrime = (n) => {
	if (n <= 1) {
		return false;
	}
	for (let i = 2; i * i <= n; i++) {
		if (n % i === 0) {
			return false;
		}
	}
	return true;
};

module.exports = IsPrime;

File: solutions/go/base/is_prime.go
// Costome function to check if a number is prime
package base

import ()

// IsPrime checks if a number is prime
func IsPrime(n int) bool {
	if n <= 1 {
		return false
	}
	for i := 2; i*i <= n; i++ {
		if n%i == 0 {
			return false
		}
	}
	return true
}